common/models/types/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Packages the lexical-model-types package.
+# Packages the @keymanapp/models-types package.
```

Comment:
Is this `build.sh` required? I see code here for publishing, though I'm not sure it matters now, with Lerna.

Reply:
We haven't started doing any publishing with Lerna - so far, we've been using it to manage versioning and within-repo links.

At minimum, we'd need to go around and add "private" markers to a lot of our packages first; most aren't publish-ready yet.  I haven't dived into all the ramifications of a `lerna publish` command yet.